data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -628,17 +632,40 @@ def _workforce_factor(self) -> bool:
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        workforce_combined_criteria_for_island_areas = (
-            self.df[unemployment_island_areas_criteria_field_name]
-            | self.df[poverty_island_areas_criteria_field_name]
-            | self.df[
-                low_median_income_as_a_percent_of_ami_island_areas_criteria_field_name
-            ]
-        ) & (
+        island_areas_high_scool_achievement_rate_threshold = (
             self.df[field_names.CENSUS_DECENNIAL_HIGH_SCHOOL_ED_FIELD_2009]
             >= self.LACK_OF_HIGH_SCHOOL_MINIMUM_THRESHOLD
         )
 
+        self.df[
```

Comment:
Actually create and store the criteria field.

Reply:
👍